torchprime/hf_models/README.md

```diff
@@ -21,4 +25,28 @@ add flag `--use-hf` to `tp run` command:
 tp run --use-hf torchprime/hf_models/train.py
 ```
 
+You may override the arguments to the Hugging Face training script via the Hydra
+config syntax. Examples:
+
+```sh
+# Run Llama 3.1 405B Hugging Face distributed training on an XPK cluster.
+tp run --use-hf torchprime/hf_models/train.py \
+    train_script.args.config_name=torchprime/hf_models/configs/model/llama-3/config_405b.json \
+    train_script.args.per_device_train_batch_size=64 \
+    train_script.args.fsdp_config=null \
+    train_script.args.fsdp=null \
+    train_script.args.tokenizer_name="meta-llama/Meta-Llama-3.1-405B" \
+    +train_script.args.spmd_2d_sharding=4
```

Comment:
could you please add a paragraph explaining how sharding would be applied for HF model

Reply:
Yes